test_scripts/API/VehicleData/GenericNetworkSignalData/025_Applying_deprecated_for_custom_data.lua

```diff
@@ -23,19 +23,19 @@ local common = require('test_scripts/API/VehicleData/GenericNetworkSignalData/co
 runner.testSettings.isSelfIncluded = false
 config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 3
 config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
-config.application2.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
-config.application2.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+config.application2.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application2.registerAppInterfaceParams.syncMsgVersion.minorVersion = 9
 
 --[[ Local Variables ]]
 local itemBool
 
 for VDkey, VDitem in pairs (common.customDataTypeSample)do
   if VDitem.name == "custom_vd_item5_boolean" then
+    itemBool = common.cloneTable(common.customDataTypeSample[VDkey])
     common.customDataTypeSample[VDkey]["since"] = "1.0"
     common.customDataTypeSample[VDkey]["until"] = "5.0"
-    itemBool = common.cloneTable(common.customDataTypeSample[VDkey])
     itemBool.deprecated = true
-    itemBool.since = "5.0"
```

Comment:
@GetmanetsIrina Question is to the case itself:
As far as I understand if parameter is deprecated since some version (e.g. `5.1` in our case) it means app with version higher (`5.9` in our case) is not able to proceed with this parameter.
However this behavior is not reflected in script

Reply:
@dboltovskyi, app is still able to proceed this parameter, it means only that this parameter is not up to date already from 5.0 version, but is still in use for backward compatibility.

Reply:
@GetmanetsIrina Thank you for clarification